future-apps/payroll/contracts/test/mocks/PayrollMock.sol

```diff
@@ -6,33 +6,12 @@ import "../../Payroll.sol";
 contract PayrollMock is Payroll {
     uint64 private _mockTime = uint64(now);
 
-    /* Ugly hack to work around this issue:
-     * https://github.com/trufflesuite/truffle/issues/569
-     * https://github.com/trufflesuite/truffle/issues/737
-     */
-    function addEmployeeShort(
-        address _accountAddress,
-        uint256 _initialDenominationSalary,
-        string _name,
-        string _role
-    )
-        external
-    {
-        _addEmployee(
-            _accountAddress,
-            _initialDenominationSalary,
-            _name,
-            _role,
-            getTimestamp64()
-        );
-    }
-
     function mockUpdateTimestamp() public { _mockTime = uint64(now); }
     function mockSetTimestamp(uint64 i) public { _mockTime = i; }
     function mockAddTimestamp(uint64 i) public { _mockTime += i; require(_mockTime >= i); }
```

Comment:
In the other apps, we've used `mock_camelCase()` as the function names for the mock functions

Reply:
That's true, but it's also true that it's kind of weird that we are mixing camel case and snake case ;-)

Reply:
Will take care of improving the way we mock time-related stuff in a separate PR :)